File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution(object):
    def getConcatenation(self, nums):
        n = len(nums)
        l = 2*n
        j=0
        ans = []
        for i in range(l):
            if j == n:
                j=0
            ans.append(nums[j])
            j+=1
        return ans
            
            
            
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        

File: 1929-concatenation-of-array/1929-concatenation-of-array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        int l = 2*n;
        int j=0;
        vector<int> ans(l);
        for(int i =0;i<l;i++)
        {
            if(j==n)
            j=0;
            ans[i]=nums[j];
            j++;
        }
        return ans;
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        
        // int n = nums.size();
        // int l = 2*n;
        // vector<int> ans(l);
        // int j =0;
        // for(int i =0;i<l;i++)
        // {
        //     if(j==n)
        //     j=0;
        //     ans[i] = nums[j];
        //     j++;
        // }
        // return ans;
    }
};